zerver/webhooks/gocd/view.py

```diff
@@ -0,0 +1,49 @@
+# Webhooks for external integrations.
+
+from typing import Any, Dict, Iterable, Optional, Text
+
+from django.http import HttpRequest, HttpResponse
+from django.utils.translation import ugettext as _
+
+from zerver.decorator import api_key_only_webhook_view
+from zerver.lib.actions import check_send_stream_message
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_error, json_success
+from zerver.lib.validator import check_dict, check_string
+from zerver.models import UserProfile
+
+MESSAGE_TEMPLATE = (
+    u'Author: {}\n'
+    u'Build status: {} {}\n'
+    u'Details: [id]({})\n'
+    u'Comment: {}'
+)
```

Comment:
Could you please use explicit keyword arguments here like `'Author: {author}\n`? Otherwise, it is a bit confusing as to what ends up where in the final message. :)